### PR TITLE
Add checksum for py-importlib-resources@5.9.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-importlib-resources/package.py [hunk 1]
```diff
@@ -12,16 +12,19 @@ class PyImportlibResources(PythonPackage):
...
     depends_on("python@2.7:2.8,3.4:", type=("build", "run"))
-
     depends_on("python@3.6:", when="@5:", type=("build", "run"))
+    depends_on("python@3.7:", when="@5.9.0:", type=("build", "run"))
+
     depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools@56:", when="@5.9.0:", type="build")
     depends_on("py-setuptools-scm@3.4.1:+toml", when="@5:", type="build")
     depends_on("py-zipp@0.4:", when="@5.0:5.1", type=("build", "run"))
     depends_on("py-zipp@3.1.0:", when="@5.2.2: ^python@:3.9", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-importlib-resources/package.py:30]: This wheel dep can be removed, all versions depend on wheel through PythonPackage

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
@spackbot run pipeline

**Comment by iarspider** [comment]
@spackbot run pipeline

**Comment by iarspider** [comment]
@spackbot run pipeline

**Comment by alalazo** [comment]
Force merged since every other PR will fail package audits

**Pull request merged**